data/testPhotoData.js

```diff
@@ -0,0 +1,74 @@
+const testPhotoData = [
+  {
+    id: 292,
+    list: 0,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'eum doloremque in',
+  },
+  {
+    id: 293,
+    list: 1,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'temporibus nihil voluptates',
+  },
+  {
+    id: 294,
+    list: 2,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'id laboriosam et',
+  },
+  {
+    id: 295,
+    list: 3,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'nihil quasi reiciendis',
+  },
+  {
+    id: 296,
+    list: 4,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'officiis ad perspiciatis',
+  },
+  {
+    id: 297,
+    list: 5,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'consectetur molestiae enim',
+  },
+  {
+    id: 298,
+    list: 6,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'sint illum quasi',
+  },
+  {
+    id: 299,
+    list: 7,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'dignissimos itaque nulla',
+  },
+  {
+    id: 300,
+    list: 8,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'iusto hic nihil',
+  },
+  {
+    id: 301,
+    list: 9,
+    roomid: 29,
+    imageurl: 'http://lorempixel.com/640/480',
+    imagedesc: 'magnam est facilis',
+  },
+];
+
+export default testPhotoData;
```

Comment:
I believe this is just test data, so don't forget to remove this later on if it's no longer relevant.